.appveyor.yml

```diff
@@ -1,45 +0,0 @@
-build: false
```

Comment:
purpose of AppVeyor was to test PHP CS Fixer under Windows too, instead of only unix. please keep AppVeyor